rlbot_gui/gui/js/botmanager.js

```diff
@@ -0,0 +1,154 @@
+var globalJson
+
+	async function downloadRepos(){
+		await fetch('https://raw.githubusercontent.com/ard1998/RLBot-repos/master/stableThrusted.json')
```

Comment:
Should it always come from here?
Should we instead have this be an rlbot controlled repo?

also is there a type in stableThrusted?

Reply:
it always was the intention to have an rlbot controlled repo with bots hosted by rlbot so we have an repo that will be safe for sure. for fast updated community repos could be added on own risk (plan)